### PR TITLE
Mustafa_Aisal conflict trial

### PR DESCRIPTION
**Pull request by MustafaOk**
con con con fli fli fli ct ct ct

### CODE REVIEW

**Review by IgorTirsina — COMMENTED** [review]
Can we remove the hard coded data from our classes :)

**Comment by VictorIachim** [comment]
> con con con fli fli fli ct ct ct

O o o orayles... auto parts...pew

**Review by VictorIachim — APPROVED** [review]
Very nice

File: src/test/java/Mustafa_Aisal.java [hunk 1]
```diff
@@ -1,8 +1,15 @@
 public class Mustafa_Aisal {
+    //Mustafa-cucumber-tomatoes-pepper-blackpepper
+    //mustafa-my dinner is ready..
     public static void main(String[] args) {
 
+        System.out.println("Conflict happen plz");
+        System.out.println("Mustafa=> As you can see praying is not a solution alone. Destroy our life Victor!");
+
+
```
Comment (IgorTirsina) [inline src/test/java/Mustafa_Aisal.java:9]: we can add comments on lines of the code to explain what it can be done, or what is wrong

**Review by IgorTirsina — APPROVED** [review]
it looks good!

**Pull request merged**